tests/stress/scopez_server/scopez_stress_test.py

```diff
@@ -0,0 +1,279 @@
+    print('| %10.2f | %10d | %10d | %10d | %10d | %10d |' % (
+        (a_num_reqs*1000/a_time_delta_ms),
+        a_results['200'],
+        a_results['300'],
+        a_results['400'],
+        a_results['500'],
+        (a_num_configs*1000/a_time_delta_ms)))
+# ------------------------------------------------------------------------------
+#
+# ------------------------------------------------------------------------------
+def get_rqst(a_host, a_id, a_vectors, a_idx, a_results):
+    l_url = a_host
+    l_v = a_vectors[a_idx]
+    l_headers = None;
+    l_body = ''
+    if 'uri' in l_v:
+        l_url = '%s/%s'%(a_host, l_v['uri'])
+    if 'query_string' in l_v:
+        l_url += '?'
+        l_url += l_v['query_string']
+    if 'headers' in l_v and len(l_v['headers']):
+        l_headers = l_v['headers']
+    if 'body' in l_v:
+        l_body = base64.b64decode(l_v['body'])
+    else:
+        l_body = l_body.encode()
+    l_r = requests.get(l_url, headers = l_headers)
+    l_code = l_r.status_code
+    if l_code >= 200 and l_code < 300:
+        if '200' in a_results:
+            a_results['200'] += 1
+        else:
+            a_results['200'] = 1
+    if l_code >= 300 and l_code < 400:
+        if '300' in a_results:
+            a_results['300'] += 1
+        else:
+            a_results['300'] = 1
+    if l_code >= 400 and l_code < 500:
+        if '400' in a_results:
+            a_results['400'] += 1
+        else:
+            a_results['400'] = 1
+    if l_code >= 500 and l_code < 600:
+        if '500' in a_results:
+            a_results['500'] += 1
+        else:
+            a_results['500'] = 1
+# ------------------------------------------------------------------------------
+#
+# ------------------------------------------------------------------------------
+def post_config(a_host, a_template, a_type, a_idx):
+    if isinstance(a_template, list):
+        for l_instance in a_template:
+            if 'last_modified_date' in l_instance:
+                l_instance['last_modified_date'] = datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+            if 'name' in a_template:
+                l_instance['name'] = str(a_idx);
+    else:
+        if "last_modified_date" in a_template:
+            a_template['last_modified_date'] = datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+        if 'name' in a_template:
+            a_template['name'] = str(a_idx);
+    l_headers = {}
+    l_headers['Content-type'] = 'application/json'
+    l_url = '%s/update_%s'%(a_host, a_type)
+    l_body = json.dumps(a_template)
+    # ------------------------------------------------------
+    # POST
+    # ------------------------------------------------------
+    try:
+        l_rq = Request(l_url, l_body.encode(), l_headers)
+        l_r = urlopen(l_rq, timeout=20.0)
+    except Exception as l_e:
+        print('error: performing POST to %s. Exception: %s' % (l_url, l_e))
+        sys.exit(1)
+    l_body = l_r.read()
```

Comment:
We need a `.decode()` here for py3.5 support

Reply:
not sure, but it was working without a decode.. but i added one